default_mail_template/models/sale_order.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def action_quotation_send(self):
+        """Inherit method for set default mail template in send mail wizard"""
+        val = super(SaleOrder, self).action_quotation_send()
+        template_id = self.env.company.default_so_mail_template.id
```

Comment:
Don't you think https://github.com/odoo/odoo/blob/15.0/addons/sale/models/sale_order.py#L908C28-L908C47 this will interfere?